Sources/Apollo/ApolloClientProtocol.swift

```diff
@@ -24,11 +24,13 @@ public protocol ApolloClientProtocol: AnyObject {
   ///   - cachePolicy: A cache policy that specifies when results should be fetched from the server and when data should be loaded from the local cache.
   ///   - queue: A dispatch queue on which the result handler will be called. Should default to the main queue.
   ///   - contextIdentifier: [optional] A unique identifier for this request, to help with deduping cache hits for watchers. Should default to `nil`.
+  ///   - context: [optional] A context that is being passed through the request chain. Should default to `nil`.
   ///   - resultHandler: [optional] A closure that is called when query results are available or when an error occurs.
   /// - Returns: An object that can be used to cancel an in progress fetch.
   func fetch<Query: GraphQLQuery>(query: Query,
                                   cachePolicy: CachePolicy,
                                   contextIdentifier: UUID?,
+                                  context: RequestContext?,
```

Comment:
The only concern I have is that this is technically a minor breaking change. Anyone who is implementing the `ApolloClientProtocol` (which I would think is usually just for mock objects in unit tests), will have to add the new parameter. It's not a huge change, but it will not be backwards compatible. 

I'm not really sure that there is a great method for deprecation of the old signatures though. So I think this falls under acceptable minor breaking changes.

Reply:
Yeah, I couldn't think of a way to do it without creating a breaking change. Glad to hear you're likely ok with it!